The problem is that if the ganeti package is installed, but this node wasn't added to the cluster yet, logrotate send an email like this every day:

Corrupt configuration file: Can't read ssconf file /var/lib/ganeti/ssconf_master_node: [Errno 2] No such file or directory: '/var/lib/ganeti/ssconf_master_node'
Aborting.

This patch takes advantage of the rotate function of the ganeti initscript which first checks if the ganeti daemon was started. If not, then it will not call `daemon-util rotate-all-logs`

--- ./doc/examples/ganeti.logrotate.in.orig
+++ ./doc/examples/ganeti.logrotate.in
@@ -7,6 +7,6 @@
         delaycompress
         sharedscripts
         postrotate
-		@PKGLIBDIR@/daemon-util rotate-all-logs
+		rc-service ganeti rotate --quiet
         endscript
 }
